distribution/docs/src/main/resources/_contents/_transformers/mime-type-mappers-contents.adoc

```diff
@@ -0,0 +1,3 @@
+
+.[[_included_mime_type_mappers]]Included Mime Type Mappers
```

Comment:
Unsure if this is fixed, but I see this:
<img width="462" alt="screenshot 2016-11-29 09 07 43" src="https://cloud.githubusercontent.com/assets/4681511/20717621/505c5600-b613-11e6-862c-11770ac43303.png">